client/app/bundles/shared/components/Tag.jsx

```diff
@@ -0,0 +1,27 @@
+//@flow
+import React from "react";
+import css from "./Tag.scss";
+
+type Props = {
+  dark?: boolean,
+  normal?: boolean,
+  label?: string,
+  id?: string,
+};
+
+export default class Tag extends React.Component<Props> {
+  constructor(props: Props);
+  super(props);
+  }
+
+  render() {
+     const {dark,normal,label,id} = this.props;
+     const labelClassNames = `${css.label} ${dark ? css.dark : ""}${normal ? css.normal : ""}`;
+
+     return (
+       <span>
+         <div className = {labelClassNames}>{label}</div>
```

Comment:
In case there are syntax problems, update to `<div className={labelClassNames}>{label}</div>`